storage/adapters/postgresql/conn_config_test.go

```diff
@@ -27,10 +27,10 @@ func Test_ConnectionConfig_String(t *testing.T) {
 				Password: "admin",
 				Host:     "localhost",
 				Port:     "5432",
-				Database: "gouth",
+				Database: "aureole",
```

Comment:
Our db name is "test". Should we rename db or rename this config to the previous version?

Reply:
I don't want to have word "gouth" in project at all. So, we will rename database. Do I need change something in code for this?

Reply:
Got it. Please rename db_name in config.yaml and docker.compose for postgresql adapter. I will rewrite some tests with old name later.